frontend/state-management/answers/samcrider.md

```diff
@@ -0,0 +1,9 @@
+## QUESTIONS
+
+### What is state in react? Why is state management important?
+
+State is dynamic data storage. Dynamic as in it allows component logic to manipulate it. State is basically a container of all the currently important data of an application. It is important to manage state in order to reduce the run time of tasks in an application. Also, components may need to share data which requires state to be managed.
```

Comment:
"It is important to manage state in order to reduce the run time of tasks in an application."

What does this mean?